.coveragerc

```diff
@@ -0,0 +1,13 @@
+[run]
+branch = True
+concurrency =
+    gevent
+    multiprocessing
+
+[report]
+precision = 1
+include =
+    ./*
+omit =
+    lib/tarantool-python/*
+    lib/msgpack-python/*
```

Comment:
Should we add `test/*` here as well? The test file test/unittest/test_lib_utils.py is in my report.

Reply:
sure, added

```
--- a/.coveragerc
+++ b/.coveragerc
@@ -11,3 +11,4 @@ include =
 omit =
     lib/tarantool-python/*
     lib/msgpack-python/*
+    test/*

```